.github/workflows/scan-pull-request.yml

```diff
@@ -0,0 +1,47 @@
+name: "Frogbot Scan Pull Request"
+
+on:
+  workflow_call:
+    inputs:
+      upstream-packages:
+        description: A whitespace separated list of Howso package dependencies (e.g., "amalgam-lang-py howso-engine-py").
+        type: string
+        required: false
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+
+  scan-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Download Artifacts
+        run: |
+          mkdir -p dist
+          pip download -r requirements-3.11.txt -d ./dist
```

Comment:
What's the potential someone sneaks something in the other requirements files? Or is that really possible? Want to make sure someone can't sneak something in one of the other requirements files and it doesn't get checked.

Reply:
Hmm, good point. Theoretically possible, though that'd be a weird thing to do and would definitely raise some eyebrows on the PR review. The `requirements-3.X.txt` files are always generated automatically, so any manual additions there would be questioned (especially if not by an employee). I'd say checking the other versions is probably not worth the extra ~12 minute build time.

Reply:
Sounds good to me.